src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelperTest.java

```diff
@@ -247,130 +255,147 @@ public void getJurCodesCollectionWithHide_ListFromString() {
         assertFalse(BulkHelper.containsAllJurCodes(null, BulkHelper.getJurCodesListFromString(jurCodes)));
         assertFalse(BulkHelper.containsAllJurCodes(new ArrayList<>(), BulkHelper.getJurCodesListFromString(jurCodes)));
 
-        assertFalse(BulkHelper.containsAllJurCodes(new ArrayList<>(), BulkHelper.getJurCodesListFromString(null)));
-        assertFalse(BulkHelper.containsAllJurCodes(new ArrayList<>(), BulkHelper.getJurCodesListFromString("")));
-        assertFalse(BulkHelper.containsAllJurCodes(new ArrayList<>(), BulkHelper.getJurCodesListFromString(" ")));
+        assertFalse(BulkHelper.containsAllJurCodes(
+                new ArrayList<>(), BulkHelper.getJurCodesListFromString(null)));
+        assertFalse(BulkHelper.containsAllJurCodes(
+                new ArrayList<>(), BulkHelper.getJurCodesListFromString("")));
+        assertFalse(BulkHelper.containsAllJurCodes(
+                new ArrayList<>(), BulkHelper.getJurCodesListFromString(" ")));
     }
 
     @Test
     public void getJurCodesCollectionWithHide_AllJurCodesAreFiltered_ReturnsEmptyString() {
-        JurCodesTypeItem JurCodesTypeItem1 = getJurCodesWithOutcome("A", "Acas conciliated settlement");
-        JurCodesTypeItem JurCodesTypeItem2 = getJurCodesWithOutcome("B", "Withdrawn or private settlement");
-        JurCodesTypeItem JurCodesTypeItem3 = getJurCodesWithOutcome("C", "Input in error");
-        List<JurCodesTypeItem> jurCodesTypeItems = new ArrayList<>(Arrays.asList(JurCodesTypeItem1, JurCodesTypeItem2, JurCodesTypeItem3));
+        JurCodesTypeItem JurCodesTypeItem1 = getJurCodesWithOutcome(
```

Comment:
Change variable name to not start with capital

Reply:
done